x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -24,24 +27,41 @@ import { TableFetchWrapper } from '../../../shared/table_fetch_wrapper';
 import { getTimeRangeComparison } from '../../../shared/time_comparison/get_time_range_comparison';
 import { OverviewTableContainer } from '../../../shared/overview_table_container';
 import { getColumns } from './get_columns';
+import { ElasticDocsLink } from '../../../shared/Links/ElasticDocsLink';
 
 type ApiResponse = APIReturnType<'GET /api/apm/services/{serviceName}/transactions/groups/main_statistics'>;
-const INITIAL_STATE = {
-  transactionGroups: [] as ApiResponse['transactionGroups'],
+
+type InitialState = ApiResponse & {
+  requestId: string;
+  transactionGroupsTotalItems: number;
+};
+
+const INITIAL_STATE: InitialState = {
+  transactionGroups: [],
   isAggregationAccurate: true,
   requestId: '',
   transactionGroupsTotalItems: 0,
+  bucketSize: 0,
 };
```

Comment:
I like what you did in https://github.com/elastic/kibana/pull/107094/files#diff-1f8752bfc4d79520010b5d507b96176f454830da1dc8e2091578f66e42007d02R25-R28 to cleanly separate the api response from the rest. Perhaps you can do the same here:

```suggestion
const INITIAL_STATE: InitialState = {
  requestId: '',
  response: {
    transactionGroups: [],
    isAggregationAccurate: true,
    transactionGroupsTotalItems: 0,
    bucketSize: 0,
  }
};
```